### ISSUE TITLE
Will you share the code and dataset?

### ISSUE THREAD

**Issue by xiaopanchen** [yomin-y/UCAD#1]
Will you share the code and dataset?

**Comment by yomin-y** [comment]
Yes, we will share the dataset. But now there are privacy issues which the faces of some identities are too clear. We are working on it.  We promise to release the dataset in this month.

**Comment by jiangliman** [comment]
when will you release your code
